generators/reference_quality/tiktoken_token_counter/config.py

```diff
@@ -0,0 +1,43 @@
+from util.configs import build_generator_function_config
+from util.enums import State, BricksVariableType, RefineryDataType, SelectionType
+from . import tiktoken_token_counter, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_generator_function_config(
+        # strapi information
+        function=tiktoken_token_counter,
+        input_example=INPUT_EXAMPLE,
+        issue_id=359,
+        tabler_icon="SortAscendingNumbers",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        type="python_function",
+        available_for=["refinery", "common"],
+        part_of_group=[
+            "reference_quality",
```

Comment:
Not sure what the group approach is. Do we always add all groups that fit? then also maybe "text_analysis".

Also is there a list (enum) or similar of all the available groups and what they should represent?

Reply:
If it is obvious, then brick can of course also be part of multiple groups, yes! Text analysis might also fit here, but I wasn't too sure but I could add it.

Reply:
I don't mind either way but depending on the decision we should have a guideline so in the future we know